entries/0verview.md

```diff
@@ -58,28 +58,21 @@ here](/submissions.html)!
 These flagship ZeroVer projects know how to get the most out of their
 zeroes.
 
-## HashiCorp Vault and Terraform
+## PuTTY
 
-<img align="left" style="padding-right: 15px" width="15%" src="/uploads/vault_logo.png">
-[HashiCorp's Vault][vault] project aims to be an enterprise secret
-management service, comprising the bedrock of a modern,
-microservice-oriented environment. And that's what makes it one of
-ZeroVer's most important adherents.
+For decades, PuTTY had been the go-to tool for professionals for creating SSH
+connections to remote hosts. PuTTY was first released in 1999 by Simon Tatham
+and has continued to resist the lure of [Semantic Versioning][semver] since.
```

Comment:
```suggestion
For decades, PuTTY had been the go-to tool for professionals for creating SSH
connections to remote hosts. PuTTY was first released in 1999 by Simon Tatham
and has continued to resist the lure of [Semantic Versioning][semver] since.
```

```suggestion
For decades, PuTTY has been the go-to tool for professional Windows users 
creating SSH connections. PuTTY was first released in 1999 by Simon Tatham
and has resisted the taboo of nonzero major versions for over 25 years.
```